workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/PublishSolutionOutlet.tsx

```diff
@@ -76,7 +75,7 @@ const PublishSolutionOutlet = () => {
 					onClick: () => alert('Preview...'),
 				}}
 				saveAsDraftProps={{
-					disabled: activeIndex < (id ? 1 : 2),
+					disabled: parsedSchema ? !parsedSchema.success : false,
```

Comment:
I think is better to have a variable for this condition, since this is used on the bottom with the `continue` button